File: Spoj/first200/candy3.py
t = int(raw_input())
q = raw_input()
for x in range(t):
    num = 0
    total = 0
    while(True):
        cur = raw_input()
        if(cur == ''):
            if(total%num == 0):
                print("YES")
            else:
                print("NO")
            break
        else:
            cur = int(cur)
            total = total + cur
            num = num + 1
    
    

File: Spoj/candy3.cpp
#include <cstdio>
#include <iostream>
#include <algorithm>

using namespace std;

int main()
{
    int T;
    scanf("%d",&T);
    for(int i = 0; i < T; i++)
    {
        long long num = 0;
        long long tot = 0;
        long long cur;
        while(cin >> cur != '\n')
        {
            num++;
            tot += cur;
        }
        if(tot%num == 0)
            cout << "YES" << endl;
        else 
            cout << "NO" << endl;
    }
    return 0;
}
